pkg/netflow/portrollup/utils.go

```diff
@@ -0,0 +1,15 @@
+package portrollup
+
+import "strconv"
+
+// PortToString convert port to string
+func PortToString(port int32) string {
+	if port >= 0 {
+		return strconv.Itoa(int(port))
+	}
+	if port == EphemeralPort {
+		return "*"
+	}
+	// this should never happen since port is either zero/positive or -1 (ephemeral port), no other value is currently supported
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Why not considering any negative number as Ephemeral, so the special case that can never happen disappear?

Reply:
> Why not considering any negative number as Ephemeral Port, so the special case that can never happen disappear?

This is actually what's happening now, since the default returned value is also `*`.

But I would favour being precise `-1` means random ephemeral port, and if needed we can't have other negative number for different meanings (just in case).